app/controllers/StripeController.scala

```diff
@@ -130,8 +132,6 @@ class StripeController(paymentServices: PaymentServices, stripeConfig: Config, c
     }
 
     createCharge.map { charge =>
-      info(s"Stripe payment successful for request id: ${request.id}, from platform ${request.platform}")
```

Comment:
I think this change should be reverted, since it doesn't look like moving this into the `thankYouUri` call will make any difference. In both cases, the calls would be executed before the user gets the redirect response, and it'd be good to keep the `thankYouUri` function as something that purely generates a URI without any unrelated side effects.